doc/src/input/potentials.md

```diff
@@ -129,3 +129,17 @@ integer.
 atoms = ["C", "C", "C", "C"]
 torsion = {k = "40 kJ/mol", delta = "120 deg", n: 4}
 ```
+## Morse potential
+
+This potential is usually used for intramolecular interaction such as bonds,
+angles or dihedrals. It is a better approximation for the vibrational 
+structure of the molecule than the Harmonic potential. It is expressed as:
+$$ V(r) = De (1 - \exp(-a (r - r_e))^2 $$
+
+The potential type keyword is `morse`, and the parameters `width` ($a$),
+ `r0` ($r_e$) and `depth` ($De$) should be provided as strings.
+
+```toml
+[[pairs]]
+
```

Comment:
Could you supply an example line for the input? For example

```toml
atoms = ["A", "B"]
# Parameters here are 'depth', 'width' and 'r0'.
morse = {depth= "40 kJ/mol", width = "3.0 A", r0 = "4.0 A"}
```
(With the right units of course)